odoo_module_migrate/migration.py

```diff
@@ -123,8 +123,11 @@ def _get_code_from_previous_branch(self, module_name, remote_name):
     def _get_migration_scripts(self):
 
         # Add the script that will be allways executed
-        self._migration_scripts.append(importlib.import_module(
-            "odoo_module_migrate.migration_scripts.migrate_allways"))
+        module = importlib.import_module(
+            "odoo_module_migrate.migration_scripts.migrate_allways")
+        migrationScript = getattr(module, "MigrationScript")()
```

Comment:
You can use `inspect.getmembers()` to get all classes from the script and treat them as migrations, so that you don't enforce the class name to be `MigrationScript`.